### DISCUSSION TITLE
Word`s cloud

### DISCUSSION THREAD

**Discussion by Nick222** [zim-desktop-wiki/zim-desktop-wiki#1799] [Ideas]
May can anyone create plugin for cloud of high frequency word in Zim-notebook?

**Comment by jaap-karssenberg** [comment]
Sorry, but without more detailed description I can only guess what you are thinking about

**Comment by Nick222** [comment]
Can Zim-wiki compile a table of the most frequently used words in a notebook and show this list to the user (or show them graphically in the form of a “word cloud”)?

Is this understandable enough (I used Google translator)?

**Comment by Nick222** [comment]
Word`s cloud can make by RedNotebook https://rednotebook.sourceforge.io/

Sample from RedNotebook site (see "Words" in left-bottom corner)

![rednotebook-2 0-1](https://user-images.githubusercontent.com/307416/86734120-672f2300-c03a-11ea-9069-b3c2ce21096b.png)

**Comment by Nick222** [comment]
Word`s cloud by R (from Ubuntu repository)
https://cran.r-project.org/web/packages/wordcloud/index.html

**Comment by Nick222** [comment]
And by Python http://amueller.github.io/word_cloud/

**Comment by Nick222** [comment]
Move to Discussion, pls...
